.mocharc.json

```diff
@@ -8,7 +8,8 @@
   "recursive": true,
   "reporter": "dot",
   "slow": 75,
-  "timeout": 10000,
+  "timeout": 20000,
```

Comment:
fyi @coda/packs I had to increase the mocha timeout since some compiler tests run slower on CI.